app/src/androidTest/java/com/pajato/argus/SearchActivityTest.kt

```diff
@@ -61,16 +62,88 @@ class SearchActivityTest : ActivityTestBase<SearchActivity>(SearchActivity::clas
         onView(withId(R.id.save_button)).check(matches(isEnabled()))
     }
 
-    @Test fun testNetworkAutoComplete() {
-        // Select the HBO network, ensure that the save button is now enabled and click on it.
-        val hint = "h"
-        val network = "HBO Go"
+    @Test fun testAllNetworkAutoCompletes() {
+        // Go through all the hints by letter and check that the adapter behaves appropriately
+        val map = getHintToNetworkCountMap()
+        for (key in map.keys) {
```

Comment:
I would be inclined to shorten this up by not using the "value" constant, and I most likely will.  Just a heads up.